pages/common/codecrafters.md

```diff
@@ -0,0 +1,16 @@
+# codecrafters
+
+> Practice writing complex software.
+> More information: <https://codecrafters.io/>.
+
+- Run tests without committing changes:
+
+`codecrafters test`
+
+- Run tests for all previous stages and the current stage without committing changes:
+
+`codecrafters test --previous`
+
+- Commit changes and submit, to move to the next stage:
+
+`codecrafters submit`
```

Comment:
```suggestion
`codecrafters submit`

- Display help:
`codecrafters help`
```

Reply:
Our cli is pretty simple. If people run help, they will get the same info as this tldr page, so not much value in list `help` here.